velox/experimental/cudf/exec/ToCudf.cpp

```diff
@@ -420,8 +432,8 @@ struct CudfDriverAdapter {
 
   // Call operator needed by DriverAdapter
   bool operator()(const exec::DriverFactory& factory, exec::Driver& driver) {
-    if (!driver.driverCtx()->queryConfig().get<bool>(
-            CudfConfig::kCudfEnabled, CudfConfig::getInstance().enabled) &&
+    if (driver.driverCtx()->queryConfig().get<bool>(
+            CudfConfig::kCudfDisabled, CudfConfig::getInstance().disabled) &&
         allowCpuFallback_) {
```

Comment:
If cudf is disabled, do we care whether cpu fallback is allowed or not? That will just make for another combination of conflicting configurations. I would say that if cudf is not enabled, the rest of the cudf parameters should be irrelevant.